core/internal/monitor/cwmetadata.go

```diff
@@ -155,8 +146,16 @@ func (cwm *CoreWeaveMetadata) Probe(ctx context.Context) *spb.EnvironmentRecord
 }
 
 // Get fetches and parses metadata from the CoreWeave instance metadata endpoint.
+//
+//gocyclo:ignore
```

Comment:
`//gocyclo:ignore` is OK on methods that are just large switch statements, but not here. This is a pretty large method and looks like parts of it need to get factored out into helpers.